scripts/pirate.py

```diff
@@ -0,0 +1,102 @@
+#Creates the Pirate translation files.
+
+import sys #To get command line arguments.
+import pirateofdoom #Contains our translation dictionary.
```

Comment:
This must be the most obscure reference to the dictionary-of-doom ever :)

Reply:
In hindsight I should've called it a "code" or "codex" instead.